### DISCUSSION TITLE
Move skin cursor size to the skin section in settings

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#21604] [Polls]
Gameplay(Skin) cursor size should be listed within skin category since usually when you change skins you adjust the cursor size to compensate for unequally sized cursors across different skins. 
Having to go to gameplay for this adds one extra step which is unnecessary and unintuitive coming from osu stable. 
I would also argue that cursor size is as much a skin parameter as a gameplay one so the category really doesn't matter. 

Hence convenience over all!

**Comment by peppy** [comment]
Skins should all have the same size cursors. If they don't it's the skin's fault.

In the future you'll be able to go into the skin editor to adjust this. This setting won't be moving.

**Reply by Inconsist** [reply]
I can agree with your reasoning, but its a reality that cursor sizes vary widely, and the menu layout from stable feels very intuitive in this regard. In my opinion the visual aspects of gameplay could be best fit in one category, cursor size and playfieldborders included. If a user wishes to change the gameplays appearance they should be able to do this in one section. Its just unintuitive that you change your skin, und then to change something else visually you have to go and search for it every time because its not with the other visual thing you just changed. Thematically to me this makes no sense. Technically they're seperated but the design would be more userfriendly if these things were moved together somehow. Please explain to me why this is wrong, or that the skin editor can just do everything in the furutre, I dont know the roadmap for what kind of functionality its supposed to have.

**Reply by peppy** [reply]
Yes you will be able to do exactly what you want in the skin editor.

**Reply by Inconsist** [reply]
Is it planned that you will be able to access the skins file structures like you used to be able to in stable? I used to skin very actively hence my interest. Currently its a bit of a pain to work with lazer since you always have to skin in stable, then import the new skin. If you notice anything is missing you always have to generate a completely new osk in stable just to reimport it to lazer again instead of just adding the file to the folder directly. Same question for beatmaps, when mapping I occasionally wanted to edit the textfiles directly, will this ever be possible again with lazers new storage structures? Sorry if this is a bit off topic btw it didnt seem worth a new thread.

**Reply by peppy** [reply]
You can drag files into the skin editor to add them (you can delete the resulting element and the file will be updated internally). More functionality will be added in the future.
